addons/mrp/models/mrp_workorder.py

```diff
@@ -663,6 +671,17 @@ class MrpWorkorderLine(models.Model):
     finished_workorder_id = fields.Many2one('mrp.workorder', 'Finished Product for Workorder',
         ondelete='cascade')
 
+    @api.model_create_multi
+    def create(self, values):
+        for line_data in values:
+            wo = self.env['mrp.workorder'].browse(line_data.get('raw_workorder_id'))
```

Comment:
raw_wokorder_id is not required, maybe you should let the create happen the check/raise